### DISCUSSION TITLE
Is 5.0-dev branch usable?

### DISCUSSION THREAD

**Discussion by Espionage724** [joomla/joomla-cms#38316] [Q&A] [answered]
I'm looking to set-up a fresh personal Joomla instance with auto-updates. I've done it for years with 3.10, and now I want to start with a newer branch. Ideally I'd prefer the branch to stay updated for a while.

Is the 5.0 branch currently usable? I have PHP 8.

**Comment by chmst** [comment, marked as answer]
I recommend starting with 4.2. This branch works with 8.0 - it will be released soon in a stable version.
